cmd/kluctl/commands/cmd_list_images.go

```diff
@@ -39,7 +38,7 @@ func (cmd *listImagesCmd) Run() error {
 		result := types.FixedImagesConfig{
 			Images: ctx.images.SeenImages(cmd.Simple),
 		}
-		if cmd.Sort {
+		if result.Images != nil && len(result.Images) > 1 {
```

Comment:
No need to check for nil or > 1 as a `nil` slice is by definition an empty slice.